### PR TITLE
cli: Add  `-json` and `-t` flags to `namespace status` command

### PR DESCRIPTION
**Pull request by Juanadelacuesta**

### CODE REVIEW

File: command/namespace_status.go [hunk 1]
```diff
@@ -24,13 +24,26 @@ Usage: nomad namespace status [options] <namespace>
...
+  ` + generalOptionsUsage(usageOptsDefault|usageOptsNoNamespace) + `
+
+  Status Specific Options:
+  
+	-json
+	  Output the latest namespace status information in a JSON format.
+  
+	-t
+	  Format and display namespace status information using a Go template.
+  `
```
Comment (jrasell) [inline command/namespace_status.go:36]: slight fix to the output formatting:

```suggestion
Status Specific Options:
  
  -json
    Output the latest namespace status information in a JSON format.
  
  -t
    Format and display namespace status information using a Go template.
`
```

File: command/namespace_status_test.go [hunk 2]
```diff
@@ -52,6 +54,7 @@ func TestNamespaceStatusCommand_Good(t *testing.T) {
 	// Create a namespace
 	ns := &api.Namespace{
 		Name: "foo",
+		//Description: "Test namespace",
```
Comment (jrasell) [inline command/namespace_status_test.go:57]: If this isn't being used, can we remove it?

**Review by jrasell — COMMENTED** [review]
Looking good, thanks!

The PR will need a changelog entry added. Also if we could move the additional tests added to use must, that would be great. An optional change would be to move all the new and current tests in the file to use must, but this isn't required on this PR. :D

File: command/namespace_status_test.go (hunk 2 above)
Comment (Juanadelacuesta) [reply command/namespace_status_test.go:57]: It can, my bad :)

**Review by jrasell — APPROVED** [review]
LGTM, thanks @Juanadelacuesta! Lets add the backport 1.5.x label before merging this in.

**Pull request merged**